### PR TITLE
Ei-167-fix-base-address-issue

### PR DESCRIPTION
**Pull request by Stephek**

### CODE REVIEW

File: src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/Jobs/JobsService.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ public async Task RefreshLegalEntities(int pageNumber, int pageSize)
             };
 
             var response = await _client.PutAsJsonAsync(
-                "/jobs",
```
Comment (MichaelYoung1981) [inline src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/Jobs/JobsService.cs:26]: Would it not be easier just to take the / from the front of /jobs?

Comment (Stephek) [reply src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/Jobs/JobsService.cs:26]: I tried that abut it didn't fix the issue.

Comment (Stephek) [reply src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/Jobs/JobsService.cs:26]: If there is no AbsolutePath then _client.BaseAddress.AbsolutePath returns "/"

Comment (Stephek) [reply src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/Jobs/JobsService.cs:26]: You are correct.  The slash can be removed but only if it is added to the base address!

**Pull request merged**